### ISSUE TITLE
Variable width

### ISSUE THREAD

**Issue by ahopkins** [beyonk-adventures/svelte-notifications#6]
Nice work on this package. Exactly what I needed.

One thing that would be nice would be the ability to control the width. `40vw` can be pretty big in some cases, and in other it would be nice to have a fixed size. Of course it can be done with a wrapper component and overriding the CSS, but it can get messy.

**Comment by antony** [comment]
@ahopkins I agree. It could be passed as an option to the notification (I'd also consider wrapping the display duration up in this options object as well). I'd be happy to accept a PR on this if you fancy having a go.

**Comment by fujeffrey1** [comment]
I was having the same issue and gave it a shot. Heres the PR: #7
